process/dataValidators/txValidator_test.go

```diff
@@ -0,0 +1,165 @@
+package dataValidators_test
```

Comment:
Should increase code coverage. Also, the path when returning true (line 73) is not tested. I think you should test all branches in IsTxValidForProcessing func.

Reply:
Done. All branches are tested now.